impl/src/main/java/eu/darkbot/impl/managers/I18n.java

```diff
@@ -57,6 +59,7 @@ public void onPluginChange(ExtensionsAPI.PluginLifetimeEvent ev) {
         ExtensionsAPI.PluginStage stage = ev.getStage();
         switch (stage) {
             case BEFORE_LOAD:
+                formatCache.clear();
```

Comment:
probs needs to be called on setLocale too, so if a language change happens it invalidates prior messages